moped-database/views/project_list_view.sql

```diff
@@ -82,6 +81,45 @@ moped_proj_components_subtypes AS (
     LEFT JOIN moped_components AS mc ON mpc.component_id = mc.component_id
     WHERE mpc.is_deleted = false
     GROUP BY mpc.project_id
+),
+
+project_district_association AS (
+
+    WITH
+    project_council_district_map AS (
+        SELECT DISTINCT
+            project_id,
+            unnest(council_districts) AS council_district
+        FROM project_geography
```

Comment:
```sql
select moped_project.project_id, moped_proj_components.project_component_id, features.id as feature_id, project_geography.geography
from moped_project
left join moped_proj_components on (moped_project.project_id = moped_proj_components.project_id)
left join features on (moped_proj_components.project_component_id = features.component_id)
left join project_geography on (moped_project.project_id = project_geography.project_id and features.id = project_geography.feature_id)
where moped_proj_components.is_deleted is false;
```

@johnclary, thank you for this comment. The path you suggested works great, but I am wondering if you may have intended to suggest a different route that doesn't include `project_geography` and `uniform_features` by extension.

I think that there is certainly a performance cost for using the `uniform_features` view, as it does pull in **all** the underlying feature storage tables when you're asking for data from specific ones. However, it's the polymorphism complication coming back up; one doesn't know at query-authoring-time which `feature_` table you're going to ultimately be accessing for each component, and it's through the process of making them all uniform and `union`ing them that allows for the query to be written as a single query and not a pair of them tied together in some sort of loop. 

I think the performance cost can be pushed into one of two places,but not avoided. It's in the application looping over the components and looking up which `feature_` table to find the geography, or it's in the `uniform_features` table which is that same loop, just unrolled and pushed down into the database.

This stuff is complicated to me and I apologize if I'm being dense and missing the point. Thank you again for your help!

Reply:
> I am wondering if you may have intended to suggest a different route that doesn't include project_geography and uniform_features by extension.
ugh, sorry for that confusing typo. i updated my comment.

my thinking, which may be flawed, was that you could bypass the `feature_` tables in this case, because (thanks to inheritance) it is the `features` table that holds all the feature IDs in one place along with their project component. so that table alone can serve as the bridget to roll up council districts into projects. 

borrowing form your SQL:
```sql
SELECT
    moped_project.project_id,
    moped_proj_components.project_component_id,
    features.id AS feature_id,
    features_council_districts
FROM
    moped_project
    LEFT JOIN moped_proj_components ON (moped_project.project_id = moped_proj_components.project_id)
    LEFT JOIN features ON (moped_proj_components.project_component_id = features.component_id)
    LEFT JOIN features_council_districts on features_council_districts.feature_id = features.id
WHERE
    moped_proj_components.is_deleted IS FALSE;
```

Reply:
Thanks again for this feedback. Commit [ead951d](https://github.com/cityofaustin/atd-moped/pull/1274/commits/ead951d74cfc7ec582a0bbac108d5d92e0c696d4) addresses it, I believe.